plugins/org.wso2.developerstudio.humantaskeditor/src/org/wso2/developerstudio/humantaskeditor/wizards/HumanTaskWizard.java

```diff
@@ -117,13 +136,13 @@ private void doFinish(String containerName, String fileName, IProgressMonitor mo
         }
         IContainer container = (IContainer) resource;
         final IFile file = container.getFile(new Path(fileName));
-        final IFile wsdlfile = container.getFile(new Path(HumantaskEditorConstants.INITIAL_WSDL_NAME));
-        final IFile cbWsdlfile = container.getFile(new Path(HumantaskEditorConstants.INITIAL_CALLBACK_WSDL_NAME));
+        final IFile wsdlfile = container.getFile(new Path(taskName + "Task.wsdl"));
+        final IFile cbWsdlfile = container.getFile(new Path(taskName + "CBTask.wsdl"));
```

Comment:
Fixed in https://github.com/wso2/devstudio-tooling-bps/commit/3b81f244b662cbce6b9e7ed2d7c1015f2edf20d0